BuildAssets/create-linux-packages.sh

```diff
@@ -52,6 +49,7 @@ fpm --version "$VERSION" \
   --depends 'libkrb5-3' \
   --depends 'zlib1g' \
   --depends 'libicu52 | libicu55 | libicu57 | libicu60 | libicu63 | libicu66' \
+  --exclude 'opt/octopus/octopuscli/Octo' \
```

Comment:
I'm a bit curious as to why we need to `Exclude Octo legacy wrapper from distribution` in this script. Is it worth retaining that comment:

> # Exclude Octo legacy wrapper from distribution

...and/or adding extra detail about why the "legacy wrapper" is still built at all?

Reply:
It's still distributed in the `.tar.gz` for direct download, for compatibility with older versions of that tarball when the command was capitalized.

Reply:
I will clarify that in a comment.

Reply:
👍